pages/blog/houdini-and-kitql.mdx

```diff
@@ -0,0 +1,218 @@
+import withArticle from '../../ui/blog/article'
+
+export const meta = {
+  title: 'Bringing the best GraphQL experience to Svelte',
+  tags: ['graphql', 'svelte'],
+  authors: ['jycouet', 'AlecAivazis'],
+  date: '2022-07-04',
+  description:
+    "It's time to unveil a massive effort by the KitQL and Houdini maintainers to join forces!",
+  image: '/blog-assets/houdini-and-kitql/2022 05 25 - KitQL & Houdini.gif',
+  thumbnail: '/blog-assets/houdini-and-kitql/thumbnail.png'
+}
+
+export default withArticle(meta)
+
+## A Tale of Two Projects
+
+[SvelteKit](https://kit.svelte.dev/) is taking the web dev community by storm by providing an excellent solution
```

Comment:
I would go with the following:

"Since its creation, SvelteKit, had a great adoption from the Svelte community, as the library providing the best developer experience to build GraphQL applications."